libcontainer/cgroups/fs/cpuacct.go

```diff
@@ -78,27 +78,41 @@ func getCpuUsageBreakdown(path string) (uint64, uint64, error) {
 	)
 
 	// Expected format:
-	// user <usage in ticks>
-	// system <usage in ticks>
+	// key1 value1
+	// key2 value2
+	// ...
 	data, err := ioutil.ReadFile(filepath.Join(path, cgroupCpuacctStat))
 	if err != nil {
 		return 0, 0, err
 	}
+
 	fields := strings.Fields(string(data))
-	if len(fields) != 4 {
```

Comment:
Wondering; so this function is now able to handle the additional fields, which adds a lot of complexity, but the additional fields are not used anywhere.

Is the problem you're trying to solve that there's _more than_ 4 fields, or is the problem that `user` and `system` are not the first two lines (which sounds unlikely)

If it's just that there's more than 4 fields, then I think the only thing needed is to change this line;

```go
if len(fields) < 4 {
```

And the error message;

```go
return 0, 0, fmt.Errorf("failure - %s is expected to have at least 4 fields", filepath.Join(path, cgroupCpuacctStat))
```

Reply:
How about now?